.vscode/settings.json

```diff
@@ -1,11 +1,13 @@
 {
-  "tslint.enable": true,
```

Comment:
Please, keep it. As I have it disabled in my global settings and I like to manage it per project.